nixos/modules/security/pam.nix

```diff
@@ -760,7 +760,9 @@ let
     };
   }));
 
-  motd = pkgs.writeText "motd" config.users.motd;
+  motd = if !(isNull config.users.motdFile)
```

Comment:
Could we reverse the if branches and negate the if condition so it's easier to read?